curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537d86bdc3dd343688fceb.md

```diff
@@ -7,108 +7,99 @@ dashedName: step-6
 
 # --description--
 
-Now you need to give each `img` element a `src` attribute. You are going to use the format `https://cdn.freecodecamp.org/curriculum/css-photo-gallery/#.jpg`, replacing `#` with the `img` element order.
-
-So your first `img` element would use `1.jpg`, your second would use `2.jpg`, and so on.
+Next, give each `img` a `src` attribute according to it's order in the document. The first `img` should have a `src` of `https://cdn.freecodecamp.org/curriculum/css-photo-gallery/1.jpg`. The rest should be the same, except replace the `1` with the number the `img` is in the document.
 
 # --hints--
 
-All ten of your `img` elements should have a `src` attribute.
+All nine of your `img` elements should have a `src` attribute.
 
 ```js
 const images = [...document.querySelectorAll('img')];
 assert(images.every(image => image.getAttribute('src')));
 ```
 
-Your first `img` element should have the `src` attribute set to `https://cdn.freecodecamp.org/curriculum/css-photo-gallery/1.jpg`.
+Your first `img` element should have the `src` attribute set to `https://cdn.freecodecamp.org/curriculum/css-photo-gallery/1.jpg` as the value.
```

Comment:
This verbiage feels clunky to me. "set to `x` as the value` is redundant, I think?



__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
As said, it's to avoid having code near punctuation. I will take a clunky sentence if that means one fewer support thread.

How about:

> Your first `img` element should have `https://cdn.freecodecamp.org/curriculum/css-photo-gallery/1.jpg` set as the `src` attribute value.